File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.cpp
class Solution {
public:
    int triangularSum(vector<int>& nums) {
        int k=0;
        while(k<nums.size()){
            for(int i=1;i<nums.size()-k;i++){
                nums[i-1]=(nums[i]+nums[i-1])%10;
            }
            k++;
        }
        return nums[0];
        }
};

File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.py
class Solution(object):
    def triangularSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        k=0
        while(k<len(nums)):
            for i in range(1,len(nums)-k):
                nums[i-1]=(nums[i]+nums[i-1])%10
            k+=1
        return nums[0]        